client/landing/gutenboarding/available-designs.ts

```diff
@@ -169,12 +169,12 @@ export const getDesignImageUrl = ( design: Design ) => {
 /**
  * Asynchronously load available design images
  */
-export function preloadDesignThumbs() {
+export function prefetchDesignThumbs() {
 	if ( typeof window !== 'undefined' ) {
 		availableDesigns.featured.forEach( ( design: Design ) => {
 			const href = getDesignImageUrl( design );
 			const link = document.createElement( 'link' );
-			link.rel = 'preload';
+			link.rel = 'prefetch';
 			link.as = 'image';
```

Comment:
`prefetch` doesn't use an `as` attribute, so you might want to remove this line.

Reply:
It says [here](https://css-tricks.com/prefetching-preloading-prebrowsing/): 

> Firefox will only prefetch resources when “the browser is idle”.